perftest/tests/load/LoadCalcLarge.cpp

```diff
@@ -0,0 +1,46 @@
+/* -*- Mode: C++; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4; fill-column: 100 -*- */
+/*
+ * Copyright the Collabora Online contributors.
+ *
+ * SPDX-License-Identifier: MPL-2.0
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+#include <perftest/PerfTest.hpp>
+
+class LoadCalcLarge : public CombinedPerfTest
+{
+public:
+    LoadCalcLarge(const std::string resultsDir, const std::string &server) :
+        CombinedPerfTest(__func__, resultsDir, server)
+    {
+    }
+
+    void runTest() {
+        startMeasurement();
+        connectAndLoad("perftest/data/new-hello-world.fodt");
```

Comment:
Some inconsistency here: the test is called "large" but the test doc is called "new", I assume it would be good to go with one term everywhere.